pymc4/distributions/tensorflow/transforms.py

```diff
@@ -0,0 +1,17 @@
+import tensorflow as tf
+from .. import abstract
+
+
+class Log(abstract.transforms.Log):
```

Comment:
Why can't we use the TFP bijectors?

Reply:
We can, but I found no reason to do that here

Reply:
Avoiding code duplication and thus bugs.

Reply:
There is some ambiguity with tfp bijectors. sometimes the do require event_dims, sometimes not:
https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/bijectors/bijector.py#L1217
https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/bijectors/power_transform.py#L94